pkg/sql/codegen/attribute/attribute.go

```diff
@@ -158,6 +160,18 @@ func NewDictionaryFromModelDefinition(estimator, prefix string) Dictionary {
 // PremadeModelParamsDocs stores parameters and documents of all known models
 var PremadeModelParamsDocs map[string]map[string]string
 
+// ExtractDocString extracts parameter documents from python doc strings
+func ExtractDocString(module ...string) {
+	cmd := exec.Command("python", "-uc", fmt.Sprintf("__import__('extract_docstring').print_param_doc('%s')", strings.Join(module, "', '")))
```

Comment:
Should we make a local call here to print the docstring, other than start a process?

Reply:
That's somewhat hard to implement because we have to import the classes to get the docstrings. I have investigated several python bindings of golang but they are either too primitive ([go-python](https://github.com/sbinet/go-python)) or not as reliable.